src/acceptance-of-delivery/test/logic.feature

```diff
@@ -4,11 +4,26 @@ Feature: Acceptance of Delivery
   Background:
     Given that the contract says
 """
-Acceptance of Delivery. "Party A" will be deemed to have completed its delivery obligations if in "Party B"'s opinion, the "Widgets" satisfies the Acceptance Criteria, and "Party B" notifies "Party A" in writing that it is accepting the "Widgets".
+## Acceptance of Delivery.
 
-Inspection and Notice. "Party B" will have 10 Business Days' to inspect and evaluate the "Widgets" on the delivery date before notifying "Party A" that it is either accepting or rejecting the "Widgets".
+"Party A" will be deemed to have completed its delivery obligations
```

Comment:
Why are the extra line breaks necessary here? Won't this make rendering problematic?